probe/docker/container.go

```diff
@@ -245,6 +248,13 @@ func (c *container) GetNodeMetadata() report.NodeMetadata {
 	return result
 }
 
+// ExtractContainerLabels returns the list of Docker container labels given a NodeMetadata from the Container topology.
+func ExtractContainerLabels(nmd report.NodeMetadata) map[string]string {
+	result := make(map[string]string)
```

Comment:
[Avoid make and new](https://docs.google.com/document/d/18UTcmy8VoecEaYNb0DrI8NZrI1uJbq-XYDEko0l5Ny4/edit), preferring struct-literal initialization when possible, i.e.

```go
result := map[string]string{}
```